src/Security/Handler/AclSecurityHandler.php

```diff
@@ -185,6 +187,13 @@ public function findObjectAcls(\Traversable $oids, array $sids = [])
         return $acls;
     }
 
+    /**
+     * NEXT_MAJOR: change signature to `addObjectOwner(MutableAclInterface $acl, ?UserSecurityIdentity $securityIdentity = null): void`.
+     *
+     * @param MutableAclInterface $acl
+     *
+     * @return void
+     */
```

Comment:
I think the same comment should be added to the next method addObjectClassAces.

And we may precise to 
- Remove the assert in `addObjectClassAces` and `updateAcl`
- Remove the `$acl instanceof MutableAclInterface` check in `addObjectOwner`

Reply:
> And we may precise to
> 
>     * Remove the assert in `addObjectClassAces` and `updateAcl`
> 
>     * Remove the `$acl instanceof MutableAclInterface` check in `addObjectOwner`

this will not force to use right thing on language level. That's why I left it as it is

Reply:
> this will not force to use right thing on language level

Sorry I don't understand.

If in the NEXT_MAJOR, we change
```
public function addObjectClassAces(AclInterface $acl, array $roleInformation = [])
```
to 
```
public function addObjectClassAces(MutableAclInterface $acl, array $roleInformation = [])
```
the assert
```
\assert($acl instanceof MutableAclInterface);
```
will be useless in next major, isn't it ?

Reply:
yes it is.


Oh... Now I understand. 
Write a comment after `assert` to remove it

Reply:
I've added `NEXT_MAJOR` where it was needed